### PR TITLE
Revise setup for GitLab

### PR DESCRIPTION
**Pull request by boshek**

### CODE REVIEW

File: setup.qmd [hunk 1]
```diff
@@ -115,36 +115,31 @@ See [brew.sh](https://brew.sh/) for installation instructions.
...
 ```
 
-Communicating with GitHub requires authentication with your GitHub account.
+Communicating with GitLab requires authentication with your GitLAb account.
 This is achieved by creating and securely storing a Personal Access Token (PAT).
-The [Managing Git(Hub) Credentials](https://usethis.r-lib.org/articles/articles/git-credentials.html) vignette in the **`usethis`** [@usethis] package has details on using PATs, but for now the following should suffice.
+Although focused more on GitHub, the [Managing Git(Hub) Credentials](https://usethis.r-lib.org/articles/articles/git-credentials.html) vignette in the **`usethis`** [@usethis] package has details on using PATs, but for now the following should suffice.
 
-Create a PAT with:
+Create a PAT with using these instructions: https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token
```
Comment (ateucher) [inline setup.qmd:140]: Should we give suggestions on scopes and expiry?

File: setup.qmd [hunk 2]
```diff
@@ -115,36 +115,31 @@ See [brew.sh](https://brew.sh/) for installation instructions.
...
 
-```{r}
-#| eval: false
-usethis::create_github_token()
-```
 
--   Give the token a descriptive name, accept the default scopes, and click "Generate Token".
 -   Copy the newly generated PAT to your clipboard.
 -   Run `gitcreds::gitcreds_set()` in R and paste in your PAT when prompted.
```
Comment (ateucher) [inline setup.qmd:144]:
```suggestion
-   Run `gitcreds::gitcreds_set("https://gitlab.com")` in R and paste in your PAT when prompted.
```

File: setup.qmd (hunk 1 above)
Comment (ateucher) [inline setup.qmd:140]:
```suggestion
Create a PAT using these instructions: https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token
```

**Review by ateucher — APPROVED** [review]
Thanks! A couple of changes and one question/suggestion

File: setup.qmd (hunk 1 above)
Comment (ateucher) [reply setup.qmd:140]: I think this is reasonable: api, read_api, read_user, create_runner, read_repository, write_repository

Comment (boshek) [reply setup.qmd]: great thanks for this.

**Pull request merged**